app/presenters/sort_option_presenter.rb

```diff
@@ -0,0 +1,36 @@
+class SortOptionPresenter
+  attr_reader :value, :key, :label
+
+  def initialize(label: label, key: key, default: false)
+    @label = label
+    @value = label.parameterize
+    @key = key
+    @default = default
+  end
+
+  def default?
+    default.present? && default
```

Comment:
Will `default` not always be present (as it's defaulted in the `initialize` if a value isn't passed in)? 

Then we could have

```
def default?
  default
end
```

or even just add `default` to `attr_reader` ?

Reply:
`@default` can be `nil`, so I prefer to define a method that explicitly returns a boolean. I could also do `@default = default || false` in the initialiser and then use an `attr_reader`, I suppose 😃

Reply:
I see your thinking. That's cool 👍